src/AppBundle/Controller/AdminController.php

```diff
@@ -588,8 +589,18 @@ public function editTaxationAction($code, Request $request)
     /**
      * @Route("/admin/settings/tags", name="admin_tags")
      */
-    public function tagsAction(Request $request)
+    public function tagsAction(Request $request, TagManager $tagManager)
     {
+        if ($request->isMethod('POST') && $request->request->has('delete')){
+            $id = $request->request->get('tag');
+            $tag = $this->getDoctrine()->getRepository(Tag::class)->find($id);
+            $tagManager->unTagAll($tag);
```

Comment:
It should be `untagAll` (the `T` is in capital letters here). 
Class methods are case-insensitive, but let's be strict 🙂